docs/settings.rst

```diff
@@ -0,0 +1,230 @@
+.. _settings:
+
+Configuring CMake Tools
+#######################
+
+CMake Tools supports a variety of settings that can be set at the user or
+workspace level via VSCode's ``settings.json`` file. This page talks about
+the available options and how they are used.
+
+Options marked with *Supports substitution* allow variable references to appear
+in their strings. See the :ref:`var-subs` section
+
+Available Settings
+==================
+
+.. _conf-cmake.buildDirectory:
+
+``cmake.buildDirectory``
+************************
+
+Specify the build directory (The root directory where ``CMakeCache.txt`` will
+be generated).
+
+- Default: ``${workspaceRoot}/build``.
+- *Supports substitution*
+
+.. _conf-cmake.installPrefix:
+
+``cmake.installPrefix``
+***********************
+
+If specified, sets a value for ``CMAKE_INSTALL_PREFIX`` when running CMake
+configure. If not, no value will be passed.
+*Note that if ``CMAKE_INSTALL_PREFIX`` is set via ``cmake.configureArgs`` or
```

Comment:
https://vector-of-bool.github.io/docs/vscode-cmake-tools/settings.html#cmake-installprefix
Isn't displayed correctly :/